### ISSUE TITLE
Keymap switch instead of macro playback

### ISSUE THREAD

**Issue by mondalaci** [UltimateHackingKeyboard/firmware#86]
Macro playback is not supported yet, and macro play actions shouldn't do anything, but when the UHK is configured according to the picture below,
![image](https://user-images.githubusercontent.com/300402/34487762-047fafd6-efd6-11e7-9f64-453ebabbd5aa.png)
 for some reason Mod+Z switches to the test keymap instead of (not) playing a macro.

**Comment by TorC8** [comment]
I can confirm this behavior.  Given that it's a macro I attempted to define, it appears to be consistently any macro playback attempt switches to the test keymap.  I have added a keymap, but not deleted any of the original kemaps.  One I did rename to a later-in-alphabet name and code, and added another earlier in alphabet.  I would wonder whether this is related to the test keymap or to its position in the keymaps array.
